src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -59,7 +59,7 @@ public class TestJsonCommand : PSCmdlet
         /// <returns>Return value is unreachable since we always rethrow.</returns>
         private static bool UnwrapException(Exception e)
         {
-            if (e is TargetInvocationException)
+            if (e is TargetInvocationException && e.InnerException != null)
```

Comment:
I think this is just easier to understand.
```suggestion
            if (e.InnerException != null && e is TargetInvocationException)
```

Reply:
Done.